src/hls-video-element.js

```diff
@@ -0,0 +1,99 @@
+import { SuperVideoElement } from 'super-media-element';
+import Hls from 'hls.js';
+
+class HlsVideoElement extends SuperVideoElement {
+  async attributeChangedCallback(attrName, oldValue, newValue) {
+    switch (attrName) {
+      case 'src': {
+        this.load();
```

Comment:
The video element doesn't automatically load a source, so I'm not a fan of adding a difference in the API at this level.